<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDialogAbrir</class>
 <widget class="QDialog" name="QDialogAbrir">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Escolha do arquivo</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="ArvoresGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Estruturas de Dados</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="BSTCheckBox">
       <property name="text">
        <string>BST</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="TRIECheckBox">
       <property name="text">
        <string>TRIE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="AVLCheckBox">
       <property name="text">
        <string>AVL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="SPLAYCheckBox">
       <property name="text">
        <string>SPLAY</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>89</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="BCheckBox">
       <property name="text">
        <string>Árvore B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="BPlusCheckBox">
       <property name="text">
        <string>Árvore B+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="OpenHashCheckBox">
       <property name="text">
        <string>Open Hash</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="arquivoLabel">
      <property name="text">
       <string>Arquivo:</string>
      </property>
      <property name="buddy">
       <cstring>diretorioLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="diretorioLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="abrirPushButton">
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QDialogAbrir</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QDialogAbrir</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
